tools/toolchains/arm.py

```diff
@@ -37,6 +37,12 @@
     "please visit https://os.mbed.com/docs/mbed-os/latest/tools/index.html"
 )
 
+UARM_TOOLCHAIN_WARNING = (
+    "Warning: We noticed that you are using Micro Arm Toolchain. "
+    "We are deprecating the use of Micro Arm Toolchain soon. "
+    "Please upgrade your environment to Arm Compiler 6 with bare metal. For more information, "
```

Comment:
What is the meaning of `upgrade your environment`? Should you not instead link a page that shows steps to use the `target.default_lib` configuration parameter?